setup.py

```diff
@@ -18,12 +18,21 @@
     ':python_version == "3.6"': ["dataclasses"],
     "lint": ["pre-commit~=1.18"],
     "docs": ["sphinx"],
+    "tests": [
+        "pytest",
+        # re: pypy: typed-ast (a dependency of mypy) fails to install on pypy
+        # https://github.com/python/typed_ast/issues/111
+        # re: win32: pytest-mypy-plugins depends on capturer, which isn't supported on
```

Comment:
@sloria @lovasoa I was a bit bummed by the lack of Windows support (I happen to be developing on Windows at the moment), but at least it's just for tests. I might raise an issue on `pytest-mypy-plugins` though, since there's no good reason to require `capturer`.

Reply:
https://github.com/typeddjango/pytest-mypy-plugins/issues/13